effect/extrinsic.test.ts

```diff
@@ -0,0 +1,72 @@
+import { assertEquals, assertObjectMatch } from "../deps/std/testing/asserts.ts";
```

Comment:
@harrysolovay Do you have a suggestion for this file location/name?

It seems to be more an integration test than a unit test.
Perhaps it would be better to create a new folder `./e2e-test/..` in the project root.

Thoughts?